integrations/huggingface-transformers/tutorials/sparsifying_bert_using_recipes.md

```diff
@@ -0,0 +1,132 @@
+
+| Recipe name | Description | EM / F1 | DeepSparse performance | Weight and Biases |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| bert-base-12layers | Bert fine-tuned on Squad | 80.927 / 88.435 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/w3b1ggyq?workspace=user-neuralmagic) |
+| bert-base-12layers_prune80 | Prune baseline model fine-tuned on Squad at 80% sparsity of encoder units | 81.372 / 88.62 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/18qdx7b3?workspace=user-neuralmagic) |
+| bert-base-12layers_prune90 | Prune baseline model fine-tuned on Squad at 90% sparsity of encoder units | 79.376 / 87.229 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/2ht2eqsn?workspace=user-neuralmagic) |
+| bert-base-12layers_prune95 | Prune baseline model fine-tuned on Squad at 95% sparsity of encoder units | 74.939 / 83.929 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/3gv0arxd?workspace=user-neuralmagic) |
+| bert-base-6layers_prune80 | Prune 6-layer model fine-tuned on Squad at 80% sparsity of encoder units | 78.042 / 85.915 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/ebab4np4?workspace=user-neuralmagic) |
+| bert-base-6layers_prune90 | Prune 6-layer model fine-tuned on Squad at 90% sparsity of encoder units | 75.08 / 83.602 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/3qvxoroz?workspace=user-neuralmagic) |
+| bert-base-6layers_prune95 | Prune 6-layer model fine-tuned on Squad at 95% sparsity of encoder units | 70.946 / 80.483 |  | [wandb](https://wandb.ai/neuralmagic/sparse-bert-squad/runs/3plynclw?workspace=user-neuralmagic) |
+
+
+## Exporting for Inference
+
+The sparsification run with the argument `--export_onnx_path` will creates an ONNX model that can be used for benchmarking with `DeepSparse` engine. You can export a model as part of the training and pruning process (as in the commands above), or after the model is pruned. 
+
+The following command evaluate a pruned model and convert it into ONNX format:
+
+```bash
+python transformers/examples/pytorch/question-answering/run_qa.py \
+  --model_name_or_path MODELS_DIR/bert-base-12layers_prune80 \
+  --dataset_name squad \
+  --do_eval \
+  --per_device_eval_batch_size 64 \
+  --max_seq_length 384 \
+  --doc_stride 128 \
+  --cache_dir cache \
+  --preprocessing_num_workers 6 \
+  --onnx_export_path MODELS_DIR/bert-base-12layers_prune80/onnx \
+```
+
+If it runs successfully, you will have the converted `model.onnx` in `MODELS_DIR/bert-base-12layers_prune80/onnx`. You can now run it in ONNX-compatible inference engines such as [DeepSparse](https://github.com/neuralmagic/deepsparse). The DeepSparse Engine is explicitly coded to support running sparsified models for significant improvements in inference performance.
+
```

Comment:
```suggestion
## Wrap-Up
```
We need a Wrap-Up area. See end of https://docs.google.com/document/d/13NXD657bOEHYCASZMYossU0_D8JXrhqXrnlXb12M-pg/edit#heading=h.j2r6323gqmzz for example